droidlet/lowlevel/hello_robot/remote/remote_hello_robot.py

```diff
@@ -111,6 +114,17 @@ def get_base_state(self):
         s = self._robot.get_status()
         return (s["base"]["x"], s["base"]["y"], s["base"]["theta"])
 
+    def get_latest_lidar_scan(self):
```

Comment:
why "latest" in "latest_lidar_scan", there is no other kind of getting at the moment